### PR TITLE
readme review for the sag-cc-builder-repos readme

### PR DESCRIPTION
**Pull request by elenatvk**

### CODE REVIEW

File: templates/sag-cc-builder-repos/README.md [hunk 1]
```diff
@@ -33,23 +35,16 @@ All supported Windows and UNIX platforms.
 
 ## Running as a standalone composite template
 
-To add product and fix repositories in Command Central:
+To add Software AG repositories to use with the Command Central Docker builder:
```
Comment (YanaSimeonova) [inline templates/sag-cc-builder-repos/README.md]: There is a pre-requisite - the credentials for empower should have been created.
So we need either the command to apply sag-cc-creds template, or  a reference to its readme ( https://github.com/SoftwareAG/sagdevops-templates/tree/master/templates/sag-cc-creds.)

File: templates/sag-cc-builder-repos/README.md [hunk 2]
```diff
@@ -33,23 +35,16 @@ All supported Windows and UNIX platforms.
...
 
 ```bash
 sagcc exec templates composite apply sag-cc-builder-repos \
-  repo.product.url=http://aquarius-bg.eur.a.sag/103oct2018_SIC \
+  repo.product.url=https://sdc.softwareag.com/dataservewebM103/repository
```
Comment (YanaSimeonova) [inline templates/sag-cc-builder-repos/README.md]: there should be \ at the end

**Comment by elenatvk** [comment]
Thanks, Yana, I updated the readme.

File: templates/sag-cc-builder-repos/README.md [hunk 3]
```diff
@@ -25,11 +25,13 @@ Use this template to add public Software AG product and fix repositories to use
 
 Create Empower credentials by running the sag-cc-creds template:
 
+```bash
 sagcc exec templates composite apply sag-cc-creds \
   credentials.username=yourusername \
   credentials.password=yourpassword \
   credentials.key=EMPOWER \
```
Comment (YanaSimeonova) [inline templates/sag-cc-builder-repos/README.md]: Elly, it should be empower, because in the next command lowercase is used

Comment (YanaSimeonova) [reply templates/sag-cc-builder-repos/README.md]: Actually , it is better to keep EMPOWER  here and change it in the next command to be capital

Comment (elenatvk) [reply templates/sag-cc-builder-repos/README.md]: Okay, fixed.

**Pull request merged**